### ISSUE TITLE
Trouble de-duplicating occurences by grid cell

### ISSUE THREAD

**Issue by sraithel** [danlwarren/ENMTools#136]
I keep getting an error when selecting to de-duplicate by grid cell from a ascii raster file while I have no issues with exact match de-duplicating. Could it be an error related to my raster files?

here is the error code >>>

Can't open !

    while executing
"::perl::CODE(0x4bcdf50)"
    invoked from within
".b5 invoke "
    invoked from within
".b5 instate !disabled { .b5 invoke } "
    invoked from within
".b5 instate pressed { .b5 state !pressed; .b5 instate !disabled { .b5 invoke } } "
    (command bound to event)

**Comment by danlwarren** [comment]
It sounds like it's having trouble finding or reading your ASCII file.  Regardless, this github repo is for the R version of ENMTools.

**Comment by sraithel** [comment]
Hi,

Thanks for your help.
I realized I was pulling the raster file from a different location.
Although, I am still having issues once I chose from the correct location,
it produced an empty .csv file (0KB). Any ideas what could be wrong with my
ascii files?

Best,
Sophia


Sophia L. Raithel

Sophia.Raithel@gmail.com

Slr2195@columbia.edu

SophiaRaithel/wixsite.com <https://sophiaraithel.wixsite.com/site/>

(202) 870-8978


>>Please check out my fundraising pag
<https://experiment.com/projects/expanding-survey-efforts-to-evaluate-the-status-of-bioko-s-threatened-monkeys>e
for my upcoming research to survey Monkeys in Central Africa!
